src/components/ModalWithForm.js

```diff
@@ -9,23 +9,27 @@ class ModalWithForm extends Modal {
   }
 
   close() {
-    this._modalForm.reset();
     super.close();
```

Comment:
* You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.

Reply:
You haven't deleted something https://github.com/DMyers5280/se_project_aroundtheus/pull/1/files#r1707972116 

* You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.